site/layouts/mobile/list.html

```diff
@@ -0,0 +1,27 @@
+{{ define "main" }}
+
+    <div class="pv6 ph2 tc-l">
+        <a class="f6 dim br-pill ph3 pv2 ma2 dib white bg-primary">Beta</a>
+            <h1 class= "f1 b">Hire on the go.</h1>
+                <h2 class="f4">No matter where you are, you can hire with enlist's iOS and Android apps.</h2>
+                    <ul class="pa4 tl ml7">
+                        <li>&check;Find what needs your attention quickly</li>
+                        <li>&check;Search for the candidates you want</li>
+                        <li>&check;Access all your information anytime, anywhere</li>
+                        <li>&check;Collaborate with your team on the go</li>
+                    </ul>
+        
+        <div class="flex flex-wrap mhn1">
+            <div class="w-100 w-50-ns ph1-ns">
+                <a href="https://apps.apple.com/app/id1146472597">
+                    <img alt="Download from the app store" src="/img/app-store-download.svg">
+                </a>
+            </div>
+            <div class="w-100 w-50-ns ph1-ns">
+                <a href="https://play.google.com/store/apps/details?id=com.enlist&utm_source=website&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1">
```

Comment:
I think these images will need some sizes specified to get them to look consistent.